test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
```

Comment:
Yeah, I recommend just removing the inner one and renaming the outer one--that way we can keep each test suite completely isolated from one another.

The only reason we might want to nest `describe` calls is if two components or "types" of test suites need to share common infrastructure--in which case the `beforeEach`/`afterEach` of the top-level `describe` could do the setup/teardown common to both sub-suites. But until we definitely have a need for that, I think we should just keep the different suites isolated from one another as much as possible. (Unfortunately, I don't think all the test suites I've written follow this principle--sorry I'm not practicing what I'm preaching... :disappointed: )